Mesh_3/doc/Mesh_3/Mesh_3.txt

```diff
@@ -759,9 +759,9 @@ protection of the 1D-features.
 <center>
 <table border=0>
 <tr>
-  <td><img border="0" height="435px" src="./no-protection.png"></td>
-  <td><img border="0" height="435px" src="./protection-box.png"></td>
-  <td><img border="0" height="435px" src="./protection-all.png"></td>
+  <td><img border="0" src="./no-protection.png"></td>
+  <td><img border="0" src="./protection-box.png"></td>
+  <td><img border="0" src="./protection-all.png"></td>
```

Comment:
... I may have mixed up the width and the height of my images. The `height` should be `472`.